src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoLineWrapCheck.java

```diff
@@ -73,6 +95,20 @@
  *   &lt;property name="tokens" value="IMPORT"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * package com.puppycrawl. // OK
+ * tools.checkstyle.checks;
+ *
+ * import java.io.*; // OK
+ * import java.lang. // violation
+ *  Boolean;
+ *
+ * import static java.math. // OK
+ * BigInteger.ZERO;
+ * </pre>
```

Comment:
Acceptable tokens for this check are 
`IMPORT, STATIC_IMPORT, PACKAGE_DEF, CLASS_DEF, METHOD_DEF, CTOR_DEF, ENUM_DEF, INTERFACE_DEF.`

Please add example for some extended tokens, e.g. for CLASS_DEF and CTOR_DEF